src/constructs/bindings.md

```diff
@@ -2,16 +2,19 @@
 title: Bindings
 ---
 
-## On the Subject of `let` and `let rec`
+## On the Subject of `let`, `let rec`, and `let mut`
 
 The `let` and `let rec` statements introduce new names into the current scope.
 In other words, they let you declare new bindings (quite similar to constants in other languages).
 
+The `let mut` statement can be used to introduce new mutable bindings into the current scope.
```

Comment:
A thought on another way we could explain this:
```suggestion
The `let`, `let mut`, and `let rec` statements introduce new names into the current scope.
In other words, they let you create variables that can be used throughout your programs.

`let` and `let rec` both create **immutable** bindings, which means they cannot be reassigned to new values once declared.

The `let mut` statement can be used to create **mutable** bindings, which means the those names can be reassigned to new values.
```

Reply:
💯 awesome!